lib/rspec/mocks/object_reference.rb

```diff
@@ -42,44 +42,67 @@ def self.anonymous_module?(mod)
     # Used when an object is passed to `object_double`.
     # Represents a reference to that object.
     #
-    # @private
+    # @param object
```

Comment:
What does `@param object` do attached to a class?  I thought this was only valid for methods...

I'm running the doc server locally and not seeing this rendered (plus it would need a description).